pom.xml

```diff
@@ -177,6 +178,12 @@
       <type>jar</type>
     </dependency>
 
+    <dependency>
```

Comment:
it's already provided by folio-spring-base, should not need to add it explicitly